src/main/java/org/jenkinsci/test/acceptance/po/Slave.java

```diff
@@ -21,7 +21,7 @@ public class Slave extends Node {
     private final String name;
 
     public Slave(Jenkins j, String name) {
-        super(j, j.getVersion().isNewerThan(new VersionNumber("2.363")) ? j.url("manage/computer/%s/", name) : j.url("computer/%s/", name));
+        super(j, j.getVersion().isNewerThan(new VersionNumber("2.363")) && j.getVersion().isOlderThan(new VersionNumber("2.391")) ? j.url("manage/computer/%s/", name) : j.url("computer/%s/", name));
```

Comment:
Perhaps it would be simpler to unconditionally use `/computer/*/` and then make `correct_redirect_after_save` just `Assume` that it is not running against 2.364–2.390 or whatever it is?

Reply:
Or have the test use something like

```java
assertThat(currentUrl().replaceFirst("^manage/", ""), equalTo(slave.url));
```

Reply:
Your first proposal makes sense and would definitely simplifies the code.